src/ctap/storage.rs

```diff
@@ -1077,7 +982,7 @@ mod test {
         // The persistent keys stay initialized and preserve their value after a reset.
         persistent_store.reset(&mut rng).unwrap();
         assert_eq!(
-            persistent_store.attestation_private_key().unwrap().unwrap(),
+            &persistent_store.attestation_private_key().unwrap().unwrap(),
```

Comment:
Because the types make it not convenient: mismatches between `[u8; N]`, `&[u8; N]`, `Vec<u8>`, and `&[u8]` (including the one below). In other words, it is possible to compare `T` and `S` without explicit conversions, while it's not for `Option<T>` and `Option<S>`. Explicitly conversion may distract from what we are actually testing.